.docs/Using-the-CLI.md

```diff
@@ -2,7 +2,7 @@
 
 ## Step 1
 
-After extracting the `.zip`, open Command Prompt, aka `cmd` (`Terminal` on **macOS** and **Linux**).
+After extracting the `.zip`, open your terminal (Command Prompt or Powershell on **Windows**, and Terminal on **macOS** and **Linux**).
```

Comment:
Removed the extra details because they introduce confusion between terminal/shell concepts. On Windows, users should use the Terminal app ideally as well, but that is not too important.

```suggestion
After extracting the `.zip` archive, open your preferred terminal.
```